nixos/modules/config/system-path.nix

```diff
@@ -41,6 +41,11 @@ let
       pkgs.zstd
     ];
 
+    defaultPackages = map (pkg: setPrio ((pkg.meta.priority or 5) + 3) pkg)
+      [ pkgs.rsync
+        pkgs.strace
+      ];
```

Comment:
Why didn't you re-add perl for now?

Reply:
I don't think perl is a commonly used tool. Who needs it can install it, like any other language or better set explicit dependencies in scripts. Like documented in the changelog.
When it would break NixOS in any major way, we would have seen it on unstable by now.

When we would have perl there, we must discuss to add python too. I think it's more popular now and present in most distros by default. But it's not best practice to install libraries on NixOS globally.

Reply:
@davidak `perl` is widely used in shell scripts, and is in most Linux distributions base package set. I do not think removing it from `systemPackages` so close to 20.09 is a good idea.

Reply:
@stigtsp thanks for your feedback!
i don't want to cause any inconveniences, but improve the quality of nixos
so i add it again, for now, and continue discussions on how nixos should be designed and used after the release